### DISCUSSION TITLE
Solución al error en instalación de Sass 👌

### DISCUSSION THREAD

**Discussion by HumilDoc71** [Launch-X-Latam/MisionFrontEnd#578] [Team Posts]
A los que batallaron hoy con la instalación de SASS, pongan estos 2 comandos en su terminal de la carpeta y ya queda:

1. npm cache clean
2. npm install node-sass@latest

**Comment by gfrodriguez** [comment]
Intente instalando "npm install sass" https://create-react-app.dev/docs/adding-a-sass-stylesheet/, el node-sass pide instalar unos requisitos https://github.com/nodejs/node-gyp#on-windows, que pesan más de 1gb

**Comment by DavidSaboredo** [comment]
Gracias compa se gano una cerveza.

**Comment by LeticiaOlivera** [comment]
Muchas gracias 😄 
![4901283](https://user-images.githubusercontent.com/106000118/204439151-5798f56f-d554-4e3d-86d2-e294321bb635.jpg)

**Comment by EzequielR01** [comment]
Muchas gracias!!
